docs/maps/editor/broadcast.md

```diff
@@ -0,0 +1,63 @@
+{.section-title.accent.text-primary}
+# Broadcasting / Presentation zones
+
+Use the broadcast feature to build speaker zones (aka. podiums) to stream your video / audio / screen to anyone
+in the corresponding listen zone.
+
+You typically use this feature to build a stage where a speaker can present to an audience.
+The stage is the **speaker zone**. Anyone walking on the stage will be able to stream their video / audio / screen.
+The audience is the **listening zone**. Anyone walking in the audience will be able to see and hear the speaker(s) in
+the speaker zone.
+
+Please note that [access to the map editor](access.md) is required to perform the following steps.
+
+#### Creating a speaker zone
+
+// TODO: make the screenshot again with a real meetup map
+<div class="row">
+    <div class="col">
+        <img src="../images/megaphone_speaker_1.png" class="figure-img img-fluid rounded" alt="" />
+    </div>
+</div>
```

Comment:
@cesar-cardinale Sorry to ask but can you make again the screenshot of this page on a "real" amphitheater (rather than on the test map)?

This will help the reader understand what is going on.

Right now, it's really hard to understand what is going to happen because the map does not match any real use case.